src/pages/[platform]/build-a-backend/data/set-up-data/index.mdx

```diff
@@ -113,11 +113,11 @@ npx ampx sandbox --outputs-format dart --outputs-out-dir lib
 
 ## Connect your application code to the data backend
 
-Once the cloud sandbox is up and running, it will also create an `amplifyconfiguration.json` file, which includes the relevant connection information to your data backend, like your API endpoint URL and API key.
+Once the cloud sandbox is up and running, it will also create an `amplify_outputs.json` file, which includes the relevant connection information to your data backend, like your API endpoint URL and API key.
 
 To connect your frontend code to your backend, you need to:
 
-1. configure the Amplify library with the Amplify client configuration file
+1. configure the Amplify library with the Amplify client configuration file (`amplify_outputs.json`)
```

Comment:
It looks like elsewhere “configuration file” is replaced with “output file”. Should it be here as well?

Reply:
it probably should, but I didn't want to add things manually for this merge